.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,15 @@
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
```

Comment:
Running on the "latest" version of an OS is a risk. We should run on a specific version, and update when needed.
We are not testing Ubuntu compatibility on a build process.